bootstrap_example/src/scss/pages/_index.scss

```diff
@@ -0,0 +1,198 @@
+.index-page {
+  &__section {
+    background-color: $primary-color;
+  }
+
+  &__main {
+    .welcome {
+      background-image: url("~images/background-1.png");
+      background-repeat: no-repeat;
+      background-position: center center;
+      background-size: cover;
+      min-height: 800px;
+
+      &__heading {
+        color: $primary-color;
+        font-size: 3rem;
+        line-height: calc(1em + 10px);
+        text-transform: uppercase;
+        max-width: 688px;
+      }
+
+      &__line {
+        background-color: $primary-bg-color;
+      }
+
+      &__sub-heading {
+        font-family: $font-merriweather;
+        color: $transparent-color;
+        max-width: 603px;
+        font-size: calc(1rem + 2px);
+        line-height: calc(1em + 10px);
+        margin-bottom: 44px;
+      }
+    }
+
+    .get-start {
+      background-color: $primary-bg-color;
+      padding-top: 127px;
+      padding-bottom: 128px;
+
+      &__heading {
+        color: $primary-color;
+        font-size: 31px;
+        line-height: 38px;
+      }
+
+      &__line {
+        background-color: $primary-color;
+        margin-bottom: 22px;
+      }
+
+      &__sub-heading {
+        max-width: 586px;
+        line-height: 24px;
+      }
+    }
+
+    .service {
+      padding-top: 127px;
+      padding-bottom: 128px;
+      &__heading {
+        color: $black-color;
+      }
+
+      &__line {
+        background-color: $primary-bg-color;
+        margin-bottom: 78px;
+      }
+
+      &__collection {
+        padding-left: 49px;
+        padding-right: 49px;
+      }
+
+      &__item {
+        margin-top: 50px;
+
+        * {
+          text-align: center;
+        }
+      }
+
+      &__icon {
+        color: $primary-bg-color;
+        font-size: 72px;
+      }
+
+      &__title {
+        font-size: 23px;
+        line-height: 29px;
+        color: $black-color;
+      }
+
+      &__content {
+        color: #6c757d;
+        line-height: 24px;
+        font-family: $font-merriweather;
+        min-height: 191px;
```

Comment:
những con số như này a thấy mình đang fix cứng hơi nhiều.
những chỗ nào mà phải set min-width, min-height to thì e có thể cân nhắc xài margin/padding với rem để control width/height của nó